rivet.sh

```diff
@@ -1,76 +1,147 @@
 package: Rivet
 version: "%(tag_basename)s"
-tag: "3.1.6-alice1"
-source: https://github.com/alisw/rivet
+tag: "rivet-3.1.8"
+source: https://gitlab.com/hepcedar/rivet.git
 requires:
+  - HepMC3
   - YODA
   - fastjet
-  - HepMC
+  - GMP
   - "Python:(?!osx)"
   - "Python-modules:(?!osx)"
   - "Python-system:(osx.*)"
 build_requires:
   - GCC-Toolchain:(?!osx)
+  - YODA
+  - Python
 prepend_path:
-  PYTHONPATH: $RIVET_ROOT/lib/python3.9/site-packages
+  PYTHONPATH: $RIVET_ROOT/lib/python/site-packages
```

Comment:
What about systems that contain multiple python versions and have a python2 default? this could break building...

Reply:
In the recipe alidist/yoda.sh, there is already a gymnastic arounfd the python site-packages that is in place.

>  # See below at build time the management towards generic path .../lib/python/site-packages
>  PYTHONPATH: $YODA_ROOT/lib/python/site-packages

i.e. 
- comments https://github.com/alisw/alidist/blame/master/yoda.sh#L33 to l.70, making the need of the line " unset PYTHON_VERSION" in yoda.sh l.32
- https://github.com/alisw/alidist/blame/master/yoda.sh#L88 to l.104 for the creation of a "generic" symlink to "python3.x" to "Python"

It was done for yoda.sh in https://github.com/alisw/alidist/commit/fa91a6b879334decaff1dfee55de6058a63820c9